wrappa/api_auth_wrappa.go

```diff
@@ -140,12 +142,17 @@ func (wrappa *APIAuthWrappa) Wrap(handlers rata.Handlers) rata.Handlers {
 			panic("you missed a spot")
 		}
 
-		if name == atc.GetAuthToken {
+		if name == atc.GetAuthToken || name == atc.GetAccessToken {
 			newHandler = auth.WrapHandler(newHandler, wrappa.getTokenValidator, wrappa.userContextReader)
 		} else {
 			newHandler = auth.WrapHandler(newHandler, wrappa.authValidator, wrappa.userContextReader)
 		}
-		wrapped[name] = auth.CSRFValidationHandler(newHandler, rejector, wrappa.userContextReader)
+		if name != atc.CheckResourceWebHook {
```

Comment:
Hmm I don't think this will be required for the actual use-case though. The `X-Csrf-Token` validation is only checked if the user has the `ATC-Authorization` cookie present. Requests made to this endpoint will have neither `Authorization` nor `ATC-Authorization`, as the endpoint has its own special auth rules (the query param).